webapp/src/modules/vendor/decentraland/BidService.spec.ts

```diff
@@ -100,17 +104,17 @@ describe('when accepting a bid', () => {
             chainId: ChainId.MATIC_MAINNET
           } as Bid
         )
+        const call = mockSendTransaction.mock.calls[0]
```

Comment:
What do you think about renaming this variable:
```suggestion
        const callParameters = mockSendTransaction.mock.calls[0]
```